pages/en/network/developing.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Developing
+---
+
+Developers are a core part of The Graph. They are responsible for both building and querying subgraphs, unlocking data for the web3 ecosystem.
+
+## Building Subgraphs
+
+Developers that build subgraphs use the [GraphQL Schema Language](https://graphql.org/learn/schema-language/) to define the data they want to make available.
+
+The Graph CLI tool streamlines subgraph creation. When you initialize your subgraph, all the default files will be created & configured. Developers can use the built-in APIs or add other libraries as dependencies as long as they are compatible with [AssemblyScript](https://github.com/AssemblyScript/assemblyscript). Subgraphs should be thoroughly tested before deployment. Developers can use [MatchStick](https://github.com/LimeChain/matchstick) to test the mapping logic in a sandboxed environment. Once your subgraph is built & tested, it can be deployed to The Graph's decentralized network. However, before anyone can begin querying it, your subgraph must be curated & indexed.
```

Comment:
I think there are some details here which are more relevant to the developer documentation, vs. the network docs.

It is important to be quite particular with definitions here:
- Subgraphs are *deployed* to a Graph Node, which then starts indexing them. That can be local, or to the studio, or indexers can start indexing subgraphs
- Subgraphs are *published* to the Graph Network, at which point indexers might choose to deploy and index them

Reply:
Thank you for the corrections! Cleaned up this section.